packages/components/src/Inject/Inject.component.js

```diff
@@ -107,6 +107,24 @@ Inject.getAll = function injectGetAll(getComponent, config) {
 	return components;
 };
 
+/**
+ * Allow `injectable` prop support in a Component.
+ * Those props will support three shapes: string, object, react element
+ * @param {function} getComponent
+ * @param {object|string|React Element} data
+ */
+Inject.getReactElement = function getReactElement(getComponent, data) {
```

Comment:
I don't understand the goal of `injectable`?
Is this like an async `Inject`?

Reply:
just to be sure do you have read the Inject.md ?
```
injectable props are props that can have three types:
* react valid element
* string (the component name wanted)
* object (the component name + props)
```